src/iks/create_acl_rules.py

```diff
@@ -0,0 +1,277 @@
+            },
+            {
+                "name" : "cf-ip-12",
+                "action" : "allow",
+                "source" : "104.16.0.0/12",
+                "destination" : "0.0.0.0/0",
+                "direction" : "inbound",
+                "protocol" : "tcp",
+                "source_port_min" : 1,
+                "source_port_max" : 65535,
+                "destination_port_min" : 443,
+                "destination_port_max" : 443
+            },
+            {
+                "name" : "cf-ip-13",
+                "action" : "allow",
+                "source" : "172.64.0.0/13",
+                "destination" : "0.0.0.0/0",
+                "direction" : "inbound",
+                "protocol" : "tcp",
+                "source_port_min" : 1,
+                "source_port_max" : 65535,
+                "destination_port_min" : 443,
+                "destination_port_max" : 443
+            },
+            {
+                "name" : "cf-ip-14",
+                "action" : "allow",
+                "source" : "131.0.72.0/22",
+                "destination" : "0.0.0.0/0",
+                "direction" : "inbound",
+                "protocol" : "tcp",
+                "source_port_min" : 1,
+                "source_port_max" : 65535,
+                "destination_port_min" : 443,
+                "destination_port_max" : 443
+            },
+            {
+                "name" : "worker-1",
+                "action" : "allow",
+                "source" : "10.10.30.0/24",
+                "destination" : "0.0.0.0/0",
+            },
+            {
+                "name" : "worker-2",
+                "action" : "allow",
+                "source" : "10.10.20.0/24",
+                "destination" : "0.0.0.0/0",
+            },
+            {
+                "name" : "worker-3",
+                "action" : "allow",
+                "source" : "10.10.10.0/24",
+                "destination" : "0.0.0.0/0",
+            },
+            {
+                "name" : "allow-all-outbound",
+                "action" : "allow",
+                "source" : "0.0.0.0/0",
+                "destination" : "0.0.0.0/0",
+                "direction" : "outbound",
+                "protocol" : "all",
+            },
+            {
+                "name" : "deny-all-inbound",
+                "action" : "deny",
+                "source" : "0.0.0.0/0",
+                "destination" : "0.0.0.0/0",
+                "direction" : "inbound",
+                "protocol" : "all",
+            },               
+        ]
+
+        return network_acl_rules
+'''
+
+
+
+    def check_network_acl(self):
+        print("Starting Network ACL Rule Check")
+        authenticator = IAMAuthenticator(self.api_key)
+        resource_service = ResourceManagerV2(authenticator=authenticator)
+        resource = resource_service.list_resource_groups(name=self.resource_group, include_deleted=False).get_result()
+        self.resource_group_id = resource["resources"][0]["id"]
```

Comment:
The resource group id is actually already requested in iks.py. It's stored in the IntegrationInfo class as resource_id. You could just pass it as a parameter to this class.